uaclient/cli.py

```diff
@@ -140,6 +140,22 @@ def detach_parser(parser):
     return parser
 
 
+def help_parser(parser):
+    """Build or extend an arg parser for help subcommand."""
+    usage = USAGE_TMPL.format(name=NAME, command="help")
+    parser.usage = usage
+    parser.prog = "enable"
+    parser._positionals.title = "Arguments"
+    parser.add_argument(
+        "name",
+        action="store",
+        nargs="*",
```

Comment:
Only allow optional one positional arg here
```suggestion
        nargs="?",
```